ui/app/models/oidc/client.js

```diff
@@ -106,7 +108,7 @@ export default class OidcClientModel extends Model {
   }
 
   // API WIP
-  @lazyCapabilities(apiPath`identity/oidc/${'name'}/provider`, 'backend', 'name') clientProvidersPath; // API is WIP
```

Comment:
Was there a ticket tracking the AIP WIP? I searched and couldn't find one, but I kind of remember there being one linked that was created in the backend? Just checking to make sure it get's closed.

Reply:
Yeah - it the backend ticket was linked as a blocker for our UI ticket to create a client's provider list view.